css/recover.css

```diff
@@ -0,0 +1,103 @@
+/* Media queries */
```

Comment:
merge this file with the style.css file and delete empty lines and replaces tabs with spaces